smsdk/client_v0.py

```diff
@@ -1069,12 +1069,13 @@ def clean_df_machine_titles(self, table, machine=None):
                     return table
 
         # Handle EF type machine names
-        if len(machine) <= 6 and machine[:3].isnumeric():
-            machine = f"'{machine}'"
+        # Commenting these out because we already are getting machine_names as string even if they are in numerics.
+        # if len(machine) <= 6 and machine[:3].isnumeric():
+        #     machine = f"'{machine}'"
```

Comment:
We should make a copy of the demo tenant for the SDK unit testing.  This way we can add a machine with the type a name which would cause this problem.